src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
+
+- Added antiAffinity (configuration)[/docs/self-hosted/administration/configuration/#userpodaffinity] for PODs with label `dev.okteto.com/affinity`
+- Add external resource icons
+- Build credential helpers
+- Remove load time on GitHub redeploy dialog
```

Comment:
what does it mean to remove the load time?

Reply:
@bgonp this https://github.com/okteto/app/pull/5607 is tagged as feature, can you elaborate a bit to know the context?

Reply:
When redeploying a resource, we've changed the way we are checking if the user has enough permissions to handle a repository and the way we are fetching the repository data so the user experience has been improved by needing less time to trigger the same action.

I guessed that `release/new-feature` was the right label to use, but I might be wrong...

Reply:
Maybe we can rephrase it to `Reduced load time when opening Github redeploy dialog` or something like that.